docs/data/frontier-squid.md

```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
 good to do the test in the
 [upstream documentation Testing the installation section](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Testing_the_installation).
 
+## Registering Frontier Squid
+
+See the [registration doc](/common/registration#new-registrations) for instructions on registering a new resource.
+(Or see the instructions for [updating existing registrations](/common/registration#updating-existing-registrations)
+if you are adding the squid service to an existing resource or resource group.)
+You will be using `template-resourcegroup.yaml` for your reference template to register various information about your
```

Comment:
Let's link directly to the `template-resourcegroup.yaml` in the topology repository

Reply:
Also, `Use template-resourcegroup.yaml...` instead of `You will be using`